packages/account/src/Sections/Security/Passkeys/passkeys.scss

```diff
@@ -15,16 +15,22 @@
             align-items: center;
             margin-block: 1.6rem;
 
+            .dc-input--error {
+                margin-bottom: 3.2rem;
```

Comment:
can we use logical properties here for new CSS atleast ?😬

Reply:
@fasihali-deriv I think we can use `margin-bottom` and `margin-top` since our application is changing from RTL and LTR not from Top to Bottom.